css/style.css

```diff
@@ -16,4 +15,149 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/*HEADER STYLES*/
+
+#karma-logo {
+  width: 25px;
+  margin-left: 50px;
+  margin-top: 50px;
+  margin-bottom: 20px;
+}
+
+nav {
+  float: right;
```

Comment:
i would recommend trying to make your `<header>` a flex box
and the `<img>` (logo) and the `<nav>` will be the flex children
then separate the flex children using `justify-content: space-between` to push them to either side of the page, the image to the left and the nav to the right
you can then apply appropriate padding to flex parent
using` float` can be quite tricky and cause a lot of headaches.